Makefile

```diff
@@ -12,7 +12,7 @@ clean:
 	$(MAKE) -C overlay clean
 	$(MAKE) -C sys-tune clean
 	rm -rf dist
-	rm sys-tune-*-*.zip
+	rm -f sys-tune-*-*.zip
```

Comment:
Force should not be required. Neither above though but that's on me.

Reply:
It is if the file does not exist, otherwise it will error out

Reply:
A - at the start of the line should be enough then.

Reply:
i'm not quite sure what you mean, can you give an example command?

Reply:
`-rm sys-tune-*-*.zip`

Reply:
ohhh, well thats handy to know, thanks